js/test/date/testdatefmt_az_Latn_AZ.js

```diff
@@ -771,7 +771,7 @@ module.exports.testdatefmt_az_Latn_AZ = {
             second: 0,
             millisecond: 0
         });
-        test.equal(fmt.format(date), "Sentyabr 2011");
+        test.equal(fmt.format(date), "sentyabr 2011");
```

Comment:
In this formatting, Sentyabr(standAlone) is correct, no?

Reply:
Not in az locale. If you look at the availableFormats for az, the yMMM and yMMMM use the M format letter, not the L, so it uses the regular spelling of the month name. It's only the MMM and MMMM formats that use the L format letter.